configdialog.cpp

```diff
@@ -62,6 +62,10 @@ BOOL ConfigDialog::command(WPARAM wparam, LPARAM lparam) {
 				m_dialog,
 				(DLGPROC) ScreenSaverPaletteCustomizeDialog
 			);
+			for (HBRUSH brush : PaletteCustomizeDialog::used_brushes) {
```

Comment:
I would really avoid trying to delete these brushes. The bottom line is that there is no evidence at all to suggest they are causing any sort of problem, and complexity cost involved with having even more meaningless code and tracking even more state _just_ to try to delete them is not even close to worth it in my eyes.